Funcky/PublicAPI.Shipped.txt

```diff
@@ -298,8 +298,6 @@ static Funcky.Extensions.ObjectExtensions.ToEnumerable<T>(this T? item, Funcky.G
 static Funcky.Extensions.ParseExtensions.ParseBooleanOrNone(this string! candidate) -> Funcky.Monads.Option<bool>
 static Funcky.Extensions.ParseExtensions.ParseByteOrNone(this string! candidate) -> Funcky.Monads.Option<byte>
 static Funcky.Extensions.ParseExtensions.ParseByteOrNone(this string! candidate, System.Globalization.NumberStyles styles, System.IFormatProvider! provider) -> Funcky.Monads.Option<byte>
-static Funcky.Extensions.ParseExtensions.ParseDateTimeOrNone(this string! candidate) -> Funcky.Monads.Option<System.DateTime>
```

Comment:
Oh we already had these overloads? We can't remove these unless we want this to be a breaking change...

Reply:
Yes for string.

Reply:
> Per discussion IRL we decided to go with the "default parameters style" for all Parse methods.

Reply:
Decision after discussion: we add the old overload Additionally to the default (if possible)